core/src/test/java/greencity/controller/SuperAdminControllerTest.java

```diff
@@ -372,4 +372,16 @@ void includeBag() throws Exception {
     void excludeBag() throws Exception {
         mockMvc.perform(patch(ubsLink + "/excludeBag/{id}", 1L)).andExpect(status().isOk());
     }
+
+    @Test
+    void deactivateTariffForChosenParamBadRequest() throws Exception {
+        mockMvc.perform(post(ubsLink + "/deactivate"))
+            .andExpect(status().isBadRequest());
+    }
+
+    @Test
+    void deactivateTariffFotChosenParam() throws Exception {
+        mockMvc.perform(post(ubsLink + "/deactivate/")
+            .param("regionsId", "1")).andExpect(status().isOk());
```

Comment:
please inject `superAdminService`
could you please check if `superAdminService.deactivateTariffForChosenParam `is calling with expected params?